modules/ROOT/pages/deployment/general/roles-permissions.adoc

```diff
@@ -0,0 +1,128 @@
+= Roles and Permissions
+:toc: right
+
+:description: With Infinite Scale, roles and permissions are separated from a system, space and sharing point of view. The way this has been implemented makes that concept not only secure but open for upcoming and extended features.
+
+== Introduction
+
+{description}  
+
+== General Info
+
+[NOTE]
+====
+* To avoid confusing the role `User` with the user account who got a role assigned, capitalisation is used. When using `User`, the system role is meant, when using `user`, the account is meant.
+
+* Roles and permissions in this document are in context of users but *not* in context of sharing.
+====
+
+=== Role Models
+
+Infinite Scale separates between global system roles and access roles for spaces. With global system roles, Infinite Scale grants permissions for the handling and lifecycle of tasks and spaces, where these permissions are *not* connected to access space content or further managing the space. The only exeption is, when a space gets created, the creator is also granted the Manager role of the space - which can, and usually is delegated.
+
+While system roles are stored with the user as meta data, space roles are stored as extended attributes in the space.
+
+Currently, ownCloud web does not provide all capabilities from the backend to users as not everything has been developed so far. Therefore you will see some restrictions to protect the system respectively to avoid confusing users.
```

Comment:
```suggestion
Currently, ownCloud web does not provide all capabilities available in the backend as not everything has been fully developed so far. Therefore you will see some restrictions to protect the system and to avoid confusing users.
```